### ISSUE TITLE
.php file does not exist error on Demo taxonomy admin page

### ISSUE THREAD

**Issue by JasonTheAdams** [piklist/piklist#7]
### Details

Piklist Version: 0.10 Beta 2

PHP Version: 7.0.15

Server OS: MacOS 10.12.4

### What was expected to happen
The page should load without errors!

### What happened instead
The `wp-admin/edit-tags.php?taxonomy=piklist_demo_type&post_type=piklist_demo` page loads with the following error:
```
Warning: File does not exist: .php in /Users/jason/Sites/WPE Starter/wp-content/plugins/piklist/includes/class-piklist.php on line 530
```

### Steps to reproduce this
Install the latest beta and go to Piklist Demo > Demo Taxonomies

**Comment by JasonTheAdams** [comment]
Related to PR #12

**Comment by donnamcmaster** [comment]
FYI, I received this error when I imported some posts using the WordPress import tool. 

![piklist-import-error](https://cloud.githubusercontent.com/assets/421360/26122578/dcb3dfd6-3a2c-11e7-8ea8-9a4457880dd9.png)

**Comment by JasonTheAdams** [comment]
Got it. Thanks! We'll test for that as we build out the PR.

**Comment by sbruner** [comment]
@JasonTheAdams -- I think this is fixed. Please test.

**Comment by JasonTheAdams** [comment]
@sbruner Not seeing the this bug anymore, but I am seeing the count warning on the Taxonomy > Layout > Field Groups sub-tab. I'm wondering if there's an issue with add-more groups that you partially fixed in #77.

I'll close this issue, but we should play around with the add-more groups more.
